apps/server/lib/lexical/server/state.ex

```diff
@@ -101,6 +101,7 @@ defmodule Lexical.Server.State do
            &Document.apply_content_changes(&1, version, event.content_changes)
          ) do
       {:ok, updated_source} ->
+        Api.diagnose(state.configuration.project, updated_source)
```

Comment:
Do we no longer need the throttler?

Or where should we place the throttler?

Reply:
We will likely need it, but I didn't want to reimplement. Let's see how it goes.